lib/bcdice/game_system/Garactier.rb

```diff
@@ -0,0 +1,499 @@
+            52 => 'オカモチ',
+            53 => '芭蕉扇',
+            54 => 'ハッピートリガー',
+            55 => '蠢く湿布',
+            56 => 'メガホン',
+            61 => 'トランシーバー',
+            62 => '好奇の鋲',
+            63 => 'スレッジハンマー',
+            64 => 'セントール',
+            65 => 'ケーブルナイト',
+            66 => '☆上級品☆',
+          }
+        ),
+        'ITM6' => DiceTable::D66Table.new(
+          'ランク６アイテム決定表',
+          D66SortType::NO_SORT,
+          {
+            11 => '禍福の勾玉',
+            12 => '祭壇',
+            13 => 'ネジまき心臓',
+            14 => 'まどろみの頭蓋',
+            15 => '猛進拍車',
+            16 => '炉心結晶',
+            21 => '狂奔の鞭',
+            22 => '暁のベル',
+            23 => 'ネコシッポ',
+            24 => '鬼蜘蛛',
+            25 => '戦上手の脚',
+            26 => '薄絹の外套',
+            31 => 'ネコクロー',
+            32 => 'クライムチャンバー',
+            33 => '古の灯火',
+            34 => 'かしこい触手',
+            35 => 'オペラグラス',
+            36 => '大鉄拳',
+            41 => '妖刀',
+            42 => 'ヘビーライター',
+            43 => '緋緋色の針',
+            44 => 'バーサクシール',
+            45 => '光芒のアンクレット',
+            46 => 'ネコブーツ',
+            51 => '旅するコイン',
+            52 => '光子鏡壁',
+            53 => 'フェザージャケット',
+            54 => 'レーザーミニオン',
+            55 => 'マニピュレーター',
+            56 => 'ランパートシールド',
+            61 => '選定者の瞳',
+            62 => '打ち上げ花火',
+            63 => '魔笛',
+            64 => '指輪',
+            65 => 'ネコミミ',
+            66 => '☆上級品☆',
+          }
+        ),
+      }.freeze()
+
+      # 施設表
+      SISETSU_TABLES = {
+        "SST" => DiceTable::Table.new(
+          "増強可能施設決定表",
+          "1D6",
+          [
+            '広場　マーケット　楽団',
+            '広場　ガレージ　鉄工所',
+            '広場　訓練場　農園　保健所',
+            '広場　学舎　骨董屋',
+            '広場　塗装工　菓子屋　貯蔵庫',
+            '広場　診療所　礼拝堂',
+          ]
+        ),
+      }.freeze
+
+      # 部位表
+      BUI_TABLES = {
+        "BUI" => DiceTable::Table.new(
+          "命中部位決定表",
+          "1D6",
+          [
+            '頭部',
+            '胴体',
+            '右腕',
+            '左腕',
+            '脚部',
+            '任意部位',
+          ]
+        ),
+      }.freeze
+
+      register_prefix('^GR[HDMS]?', 'ITM[2-6]', 'BUI', 'SST')
+
+      def eval_game_system_specific_command(command)
+        cmd_gr(command) ||
+          roll_item(command) ||
+          roll_tables(command, BUI_TABLES) ||
+          roll_tables(command, SISETSU_TABLES)
+      end
+
+      # GR系コマンドの分割
+      def cmd_gr(command)
+        case command
+        when /^GRS/
+          roll_search(command)
+        when /^GR[HDM]/
+          roll_target(command)
+        when /^GR/
+          roll_gr(command)
+        end
+      end
+
+      # 探索・索敵判定
+      def roll_search(command)
+        m = %r{^GRS([+-/*\d]+)?$}.match(command)
+        unless m
+          return nil
+        end
+
+        if m[1]
+          modifier = Arithmetic.eval(m[1], RoundType::FLOOR)
+        else
+          modifier = 0
+        end
```

Comment:
Rubyではifも式で、戻り値を持つので以下のような書き方が適切かと思います。

```suggestion
        modifier = if m[1]
                     Arithmetic.eval(m[1], RoundType::FLOOR)
                   else
                     0
                   end
```
他の部分も同様に修正をお願いします。

Reply:
もしくは、以下のようにしてしまうのも手です。

```ruby
modifier = Arithmetic.eval(m[1] || "", RoundType::FLOOR) || 0
```

Reply:
承知しました急ぎ修正いたします

Reply:
修正、マージいたしました。（2つ目の案をいただきました）
ありがとうございます！